static/queues.html

```diff
@@ -24,11 +24,19 @@ <h2>
     <aside></aside>
     <main>
       <section class="card">
+        <div id="multiselect-controls" class="popup-card">
```

Comment:
```suggestion
        <div id="multiselect-controls" class="popup-card hide">
```

Accidentally removed during extraction to CSS class.